pennylane/ops/identity.py

```diff
@@ -92,9 +71,11 @@ def compute_matrix():  # pylint: disable=arguments-differ
         """
         return np.eye(2)
 
-    @staticmethod
-    def compute_sparse_matrix(*params, **hyperparams):
-        return sparse.csr_matrix([[1, 0], [0, 1]])
+    def matrix(self, wire_order=None):
```

Comment:
May need to manually specify `has_matrix=True`.  At least if we are able to get rid of `compute_matrix`, then we need it.

Reply:
Should we also do this for all op. arithmetic classes?